### PR TITLE
Processing module's main function

### PR DESCRIPTION
**Pull request by dsblv**
My attempt to solve issue #8

Option to prevent function from processing is `excludeMain`. `false` by default.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -95,6 +95,14 @@ Type: `array`
 
 Pick which methods in a module **not** to promisify.
 
+##### excludeMain
+
+Type: `array`
+
+*Works for `pify.all()` only.*
+
+Set to `true` if you do **not** want to promisify module's main function.
```
Comment (sindresorhus) [inline readme.md]: Maybe show an example or mention a use-case. I don't think most realize you can have `main` method with additional methods a properties.

**Comment by dsblv** [comment]
Sounds reasonable.

**Comment by dsblv** [comment]
It would be much better if example was about existing and popular module, but I couldn't recall any, so I ended up with dummy module in readme :(

**Comment by sindresorhus** [comment]
Looks excellent. Thank you! :)

**Comment by dsblv** [comment]
Thank you for adding me to this repo, really appreciate that!

**Comment by sindresorhus** [comment]
I appreciate your excellent improvements :)

**Pull request closed without merging**